### PR TITLE
Fix adding a bug to multiple alerts at once

### PR DESCRIPTION
**Pull request by mishravikas**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -325,7 +325,8 @@ def run_addfields_data():
         typeVal = query_dict.pop('type')
 
     if typeVal == "bug":
-        sql = "update alerts set bug='%s',status='Investigating' where id=%s;" %(data['BugID'], data['id'])
+        ids = data['id'][:-1]
+        sql = "update alerts set bug='%s',status='Investigating' where id IN(%s);" %(data['BugID'], ids)
```
Comment (jmaher) [inline server.py]: using the IN operator, do we need to add any type of quotes around the %s ?
Also what if data['id'] == '' ?

File: js/alerts.js [hunk 2]
```diff
@@ -385,20 +385,21 @@ function performAction() {
         else if (action == "Add Bug") {
             var BugID = prompt("Please enter Bug ID");
             console.log('BUG ID-'+BugID);
+            var alertids = '';
+            for (id in checkedIds) {
+                alertids = alertids + checkedIds[id] + ",";
+            }
```
Comment (jmaher) [inline js/alerts.js]: here you could use the .join() method (http://www.w3schools.com/jsref/jsref_join.asp)

Comment (mishravikas) [inline js/alerts.js]: yeah that's perfect, thanks for it :)

File: server.py (hunk 1 above)
Comment (mishravikas) [inline server.py]: I don't think we need any quotes around %s, its working fine without them .
I assume by data['id'] == '' you mean if no alerts are selected, in that case post request will not be made https://github.com/jmaher/alert_manager/blob/master/js/alerts.js#L330

**Comment by jmaher** [comment]
deployed!

**Comment by jmaher** [comment]
this isn't working for me, specifically I am going to:
http://alertmanager.allizom.org:8080/alerts.html?rev=428d4d3c8588&showAll=1&testIndex=0&platIndex=0

the last 3 alerts in the list that are of status "Investigating", I want to mark as "Wontfix", but only the first one is marked.  I have restarted the server after deploying the changes.

**Comment by jmaher** [comment]
ah, this is only for the action "Add Bug", so we just need to fix the id field to always use the alertsids

**Comment by mishravikas** [comment]
Yes, this is only for adding bugs, and we have to do more than just changing to use the id field, have to change some things in alerts.js as well, but it won't be tough. I will open up a pr in a couple of hours.

**Comment by jmaher** [comment]
Thanks.  We will make it workable for the short term.

Joel

**Pull request merged**